src/include/connection.h

```diff
@@ -268,6 +268,18 @@ struct __wt_connection_impl {
 	uint64_t  ckpt_time_recent;	/* Checkpoint time recent/total */
 	uint64_t  ckpt_time_total;
 
+	/* Checkpoint stats and verbosity timers */
+	struct timespec ckpt_start_time_after_scrub;
+	struct timespec ckpt_stop_time;
+	struct timespec ckpt_verb_start_time;
```

Comment:
maybe we should call it ckpt_start_time_before_scrub. When I am reading code where we use ckpt_verb_start_time, I have to scroll down to these declarations to figure if this one starts before or after scrub phase.

Reply:
The purpose of this time is to display time delta from checkpoint start, in verbose messages. Renaming it to ckpt_start_time_before_scrub might not imply the reason behind using this time. I can rename it to ckpt_verb_start_time_before_scrub if that sounds OK.